docs/docs/dev_docs/contracts/syntax/events.md

```diff
@@ -122,6 +122,15 @@ aztec-cli get-logs --from 5 --limit 1
 </TabItem>
 </Tabs>
 
+Get logs functionality provides a variety of filtering options.
+To display them run:
+
+```bash
+aztec-cli get-logs --help
+```
+
+or check the definition of `LogFilter` type.
```

Comment:
What's the `LogFilter` type? Where can I find it?

Reply:
Maybe we can include this in the generated API reference for aztecjs and link to it? If not, I'd just remove the mention to `LogFilter`.

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/2600/commits/39e1de03136716a634e35e5016bb7bf2aeb5f516